teambase_description/urdf/teambase.xacro

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='utf-8'?>
+<robot name="TeamBase" xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:property name="robot_namespace" value="TeamBase"/>
+
+  <link name='${robot_namespace}/base_link'>
```

Comment:
change `base_link` to `link` to match the actual link name of the TeamBase model?

Reply:
Fixed in 32252498f9252f310d06120ffc17673e307b2e24